packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/angular/sign-in-form.md

```diff
@@ -0,0 +1,60 @@
+Build a sign-in page that captures both the username and password. As an example, from the sample application, see the component files such as `menu.component.ts`:
```

Comment:
This should be showing the app.component.ts/html instead (and maybe dynamic-form.component). IMO, the `MenuComponent` doesn't need to be covered at all.

Also, there probably need to be references to handling the `NextStep` response to convert those into form fields. This is done in `dynamic-form.component` and `form-transformer.service.ts`.

Reply:
Thanks, removed MenuComponent bit and added code snippets for `app.component.ts` and `app.component.html`. Added a small paragraph commenting on the dynamic form and the two files you mentioned, as well as links to the Interaction Code grant and the `nextStep` doc in the SDK.